packages/features/ee/teams/pages/availability.tsx

```diff
@@ -54,11 +54,11 @@ export function TeamAvailabilityPage() {
       <LicenseRequired>
         {!!errorMessage && <Alert className="-mt-24 border" severity="error" title={errorMessage} />}
         {isLoading && <Loader />}
-        {isFreeUser ? (
+        {/* {isFreeUser ? (
```

Comment:
This is technically free now right @PeerRich We don't need to gate this anymore. We should implement team billing properly tho.